server/services/educationAndWorkPlanService.test.ts

```diff
@@ -124,7 +143,23 @@ describe('educationAndWorkPlanService', () => {
       expect(educationAndWorkPlanClient.getGoalsByStatus).toHaveBeenCalledWith(prisonNumber, status, userToken)
       expect(actual).toEqual(expectedResponse)
     })
+    it('should return a problem loading the data if prison ids could not be loaded with status 404', async () => {
```

Comment:
As above, not sure this is a scenario that should stop the core data being displayed on the page 🤔 

Also, is this scenario even possible? Why would the prisonService return a 404? The prisonRegisterClient would return a 404 to the prisonService, but does (should?) the service not handle that error in some way? 🤔